packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -21,12 +21,6 @@ import { Language } from './lang/Language';
 
 const log = Log.module('MonacoUtils');
 
-window.MonacoEnvironment = {
-  getWorker(workerId, label) {
-    return editorWorker();
-  },
-};
-
```

Comment:
I'm guessing you removed this b/c of TS. You can use this instead to keep worker support. Monaco bumped their types at some point after 0.31 I think to extend window

```ts
declare global {
  interface Window {
    MonacoEnvironment: monaco.MonacoEnvironment;
  }
}

window.MonacoEnvironment = {
  getWorker() {
    return editorWorker();
  },
};
```